### PR TITLE
Update index.rst to include MAD info

### PR DESCRIPTION
**Pull request by johnsolk**
Might need to update time/day if people object to Wednesdays (spring quarter 2018)

### CODE REVIEW

File: index.rst
Comment (karenword) [inline index.rst]: I'm a little concerned about confusion/overlap with the DSI community. Maybe just "events" here?

File: index.rst
Comment (karenword) [inline index.rst]: As much as I would like to say "eavesdrop" (and I probably suggested this) I think it is maybe more diplomatic to say "listen in on problem-solving sessions to gain inspiration"

**Comment by johnsolk** [comment]
Thanks @karenword, modified wording!

**Review by raynamharris — APPROVED** [review]

**Comment by ctb** [comment]
(@raynamharris @karenword @ljcohen question - do you not have merge privileges?)

**Comment by johnsolk** [comment]
I do. But thought it was against protocol to merge my own PR?

**Comment by raynamharris** [comment]
I have merge privileges. I thought I would first make an approval. Next time I'll approve and merge so that you don't have to merge your own PR.

**Comment by ctb** [comment]
as long as it's been looked at by a second brain, we're good to go!

**Comment by ctb** [comment]
once reviewed => merge!!

**Comment by johnsolk** [comment]
Cool, will merge next time after review. Thanks!

**Pull request merged**
